navigation/presentation.html

```diff
@@ -18,8 +18,1169 @@
     <div class="reveal">
```

Comment:
please change links for css/js files to the "https://rolling-scopes-school.github.io/react-native-course/reveal-js/blabla" it can be helpful to be on develop wave all time

Reply:
Also please change name module in title here: <title>react-native-course - name module</title> and here <meta name="description" content="rolling-scopes-school/react-native-course - react module" />

Reply:
> please change links for css/js files to the "https://rolling-scopes-school.github.io/react-native-course/reveal-js/blabla" it can be helpful to be on develop wave all time. Find example in the react PR

it seems that I don't have right to push code in the develop branch directly. @Dzmitry-Klokau could you help with this?

Reply:
> Also please change name module in title here: <title>react-native-course - name module</title> and here

done

Reply:
> please change links for css/js files to the "https://rolling-scopes-school.github.io/react-native-course/reveal-js/blabla" it can be helpful to be on develop wave all time. Find example in the react PR

done